test/integration/CommunityEndpoints.test.js

```diff
@@ -0,0 +1,246 @@
+            assert.strictEqual(res.state, 'ACCEPTED')
+            assert.strictEqual(res.memberAddress, memberWallet.address)
+            assert.strictEqual(res.communityAddress, community.address)
+
+            // too much bother to check this in a separate test...
+            const res2 = await memberClient.getMemberStats(community.address)
+            assert.deepStrictEqual(res2, {
+                address: memberWallet.address,
+                earnings: '0',
+                recordedEarnings: '0',
+                withdrawableEarnings: '0',
+                frozenEarnings: '0'
+            })
+
+            // add revenue, just to see some action
+            const opWallet = new Wallet('0x5e98cce00cff5dea6b454889f359a4ec06b9fa6b88e9d69b86de8e1c81887da0', testProvider)
+            const opToken = new Contract(adminClient.options.tokenAddress, Token.abi, opWallet)
+            const tx = await opToken.mint(community.address, utils.parseEther('1'))
+            const tr = await tx.wait(2)
+            assert.strictEqual(tr.events[0].event, 'Transfer')
+            assert.strictEqual(tr.events[0].args.from, '0x0000000000000000000000000000000000000000')
+            assert.strictEqual(tr.events[0].args.to, community.address)
+            assert.strictEqual(tr.events[0].args.value.toString(), '1000000000000000000')
+            await sleep(1000)
+
+            // note: getMemberStats without explicit address => get stats of the authenticated StreamrClient
+            let res3 = await memberClient.getMemberStats(community.address)
+            while (!res3.withdrawableBlockNumber) {
+                await sleep(4000)
+                res3 = await memberClient.getMemberStats(community.address)
+            }
+            assert.deepStrictEqual(res3, {
+                address: memberWallet.address,
+                earnings: '1000000000000000000',
+                recordedEarnings: '1000000000000000000',
+                withdrawableEarnings: '1000000000000000000',
+                frozenEarnings: '0',
+                withdrawableBlockNumber: res3.withdrawableBlockNumber,
+                proof: ['0xb7238c98e8baedc7aae869ecedd9900b1c2a767bbb482df81ef7539dbe71abe4']
+            })
+
+            const isValid = await memberClient.validateProof(community.address, {
+                provider: testProvider
+            })
+            assert(isValid)
+
+            const walletBefore = await opToken.balanceOf(memberWallet.address)
+
+            const tr2 = await memberClient.withdraw(community.address, {
+                provider: testProvider
+            })
+            assert.strictEqual(tr2.logs[0].address, adminClient.options.tokenAddress)
+
+            const walletAfter = await opToken.balanceOf(memberWallet.address)
+            const diff = walletAfter.sub(walletBefore)
+            assert.strictEqual(diff.toString(), res3.withdrawableEarnings)
+        }, 600000)
+
+        // TODO: test withdrawTo, withdrawFor, getBalance
```

Comment:
Should these be done in this PR?

Reply:
made another ticket for that https://streamr.atlassian.net/browse/CPS-85

Now I'd just very much like to get these into master